polkadot/runtime/parachains/src/paras/mod.rs

```diff
@@ -1491,6 +1611,25 @@ impl<T: Config> Pallet<T> {
 		T::DbWeight::get().reads_writes(1 + pruning_tasks_done, 2 * pruning_tasks_done)
 	}
 
+	/// This function removes authorizations that have expired,
+	/// meaning their `expire_at` block is less than or equal to the current block (`now`).
+	fn prune_expired_authorizations(now: BlockNumberFor<T>) -> Weight {
+		let mut weight = T::DbWeight::get().reads(1);
```

Comment:
There's some `WeightMeter` type that does this a bit nicely, but not a big deal.